pkg/plumber/types.go

```diff
@@ -86,41 +54,6 @@ type PlumberJobSpec struct {
 	// MaxConcurrency restricts the total number of instances
 	// of this job that can run in parallel at once
 	MaxConcurrency int `json:"max_concurrency,omitempty"`
```

Comment:
`MaxConcurrency` where is this used? This is specific to prow's controllers. Why do you still want to keep this?

Reply:
I figured at some point we may wanna add a concurrently limit - but lemme zap for now